### PR TITLE
SUBMARINE-878. Modify E2E LoginIT with page object

### PR DESCRIPTION
**Pull request by noidname01**
### What is this PR for?
Modify E2E LoginIT with page object

### What type of PR is it?
[Improvement]

### What is the Jira issue?

https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-878

### How should this be tested?

```bash
# Step1: Run Submarine workbench on 127.0.0.1:8080 by operator
# Step2: Run testcase
cd submarine-cloud-v2

# Usage: ./hack/run_frontend_e2e.sh ${testcase}
# Example: 
./hack/run_frontend_e2e.sh notebookIT
```
### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

**Comment by noidname01** [comment]
@kevin85421

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/workspaceIT.java [hunk 1]
```diff
@@ -52,12 +53,11 @@ public void workspaceNavigation() throws Exception {
     String URL = getURL("http://127.0.0.1", 8080);
     Sidebars sidebars = new Sidebars(URL);
```
Comment (kevin85421) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/workspaceIT.java:55]: empty line

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/interpreterIT.java [hunk 2]
```diff
@@ -48,13 +49,14 @@ public static void tearDown(){
   @Test
   public void workspaceNavigation() throws Exception {
     String URL = getURL("http://127.0.0.1", 8080);
+
     Sidebars sidebars = new Sidebars(URL);
+
```
Comment (kevin85421) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/interpreterIT.java:54]: empty line

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/departmentIT.java [hunk 3]
```diff
@@ -51,11 +52,13 @@ public void dataNavigation() throws Exception {
...
 
     // Login
-    Login();
+    LoginPage loginPage = new LoginPage();
+    loginPage.Login();
 
     // Routing to department page
     sidebars.gotoDepartment();
 
+
```
Comment (kevin85421) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/departmentIT.java:61]: empty line

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/datadictIT.java [hunk 4]
```diff
@@ -47,14 +48,14 @@ public static void tearDown(){
...
+    LoginPage loginPage = new LoginPage();
 
     // Login
     LOG.info("Login");
-    pollingWait(By.cssSelector("input[ng-reflect-name='userName']"), MAX_BROWSER_TIMEOUT_SEC).sendKeys("admin");
-    pollingWait(By.cssSelector("input[ng-reflect-name='password']"), MAX_BROWSER_TIMEOUT_SEC).sendKeys("admin");
-    clickAndWait(By.cssSelector("button[class='login-form-button ant-btn ant-btn-primary']"));
-    pollingWait(By.cssSelector("a[routerlink='/workbench/experiment']"), MAX_BROWSER_TIMEOUT_SEC);
+    loginPage.Login();
+
```
Comment (kevin85421) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/datadictIT.java:59]: empty line

**Review by kevin85421 — CHANGES_REQUESTED** [review]

**Review by kevin85421 — APPROVED** [review]
LGTM

**Pull request closed without merging**